diff -Nru gretl-0.96/configure gretl-0.96-new/configure
--- gretl-0.96/configure	Thu Jul 19 22:07:16 2001
+++ gretl-0.96-new/configure	Mon Nov 12 14:16:03 2001
@@ -6780,7 +6780,9 @@
 else
   echo "$as_me: failed program was:" >&5
 cat conftest.$ac_ext >&5
-ac_cv_lib_readline_readline=no
+# There is no 'else' - whe have readline. PERIOD.
+# was: ac_cv_lib_readline_readline=no
+ac_cv_lib_readline_readline=yes
 fi
 rm -f conftest.$ac_objext conftest$ac_exeext conftest.$ac_ext
 LIBS=$ac_check_lib_save_LIBS
@@ -6792,7 +6794,6 @@
   CLI_LIBS="-lreadline -ltermcap" ; cat >>confdefs.h <<\EOF
 #define HAVE_READLINE 1
 EOF
-
 fi
 
 if test "$have_readline" = yes; then
@@ -6841,7 +6842,9 @@
 else
   echo "$as_me: failed program was:" >&5
 cat conftest.$ac_ext >&5
-ac_cv_lib_readline_rl_completion_matches=no
+# NO WAY!! Don't test that!
+# was: ac_cv_lib_readline_rl_completion_matches=no
+ac_cv_lib_readline_rl_completion_matches=yes
 fi
 rm -f conftest.$ac_objext conftest$ac_exeext conftest.$ac_ext
 LIBS=$ac_check_lib_save_LIBS
@@ -6853,7 +6856,6 @@
   CLI_LIBS="-lreadline -ltermcap" ; cat >>confdefs.h <<\EOF
 #define NEW_READLINE 1
 EOF
-
 fi
 
 echo "$as_me:6859: checking for inet_addr in -lc" >&5
