diff -urN gretl-0.96/config.h.in gretl-0.96-improved/config.h.in
--- gretl-0.96/config.h.in	Wed May  2 17:20:18 2001
+++ gretl-0.96-improved/config.h.in	Sat Nov 17 16:19:34 2001
@@ -21,7 +21,7 @@
 #define CONFIG_H
 
 /* Location of shared gretl files */
-#define GRETL "/usr/local/share/gretl"
+#define GRETL "%{_datadir}/gretl"
 
 /* Define if you want GNU readline support */
 #undef HAVE_READLINE
diff -urN gretl-0.96/share/Makefile gretl-0.96-improved/share/Makefile
--- gretl-0.96/share/Makefile	Sat Nov 17 16:18:20 2001
+++ gretl-0.96-improved/share/Makefile	Sat Nov 17 16:17:42 2001
@@ -33,4 +33,4 @@
 installdirs:
 	$(tooldir)/mkinstalldirs $(DESTDIR)$(gretldir) $(DESTDIR)$(gretldir)/data 
 	$(tooldir)/mkinstalldirs $(DESTDIR)$(gretldir)/data/greene
-	$(tooldir)/mkinstalldirs $(DESTDIR)$(gretldir)/scripts $(gretldir)/db
+	$(tooldir)/mkinstalldirs $(DESTDIR)$(gretldir)/scripts $(DESTDIR)$(gretldir)/db
