--- gretl-1.2.4/doc/commands/xsltrans.c.orig	2005-09-21 18:08:32.000000000 +0200
+++ gretl-1.2.4/doc/commands/xsltrans.c	2005-09-21 18:10:23.000000000 +0200
@@ -101,7 +101,7 @@
 
     /* make plain text "hlp" output */
     if (output == OUTPUT_BOTH || output == OUTPUT_HLP) {
-	full_fname("gretltxt.xsl", docdir, styname);
+	full_fname("gretltxt_cli.xsl", docdir, styname);
 	style = xsltParseStylesheetFile(styname);
 	if (style == NULL) {
 	    err = 1;
@@ -120,6 +120,13 @@
 		}
 		xmlFreeDoc(result);
 	    }
+	    xsltFreeStylesheet(style);
+	}
+	full_fname("gretltxt_gui.xsl", docdir, styname);
+	style = xsltParseStylesheetFile(styname);
+	if (style == NULL) {
+	    err = 1;
+	} else {
 	    /* gui version */
 	    result = xsltApplyStylesheet(style, doc, gui_params);
 	    if (result == NULL) {
