diff -Nru gretl-0.96/gretl_sh.in gretl-0.96-new/gretl_sh.in
--- gretl-0.96/gretl_sh.in	Fri Dec  8 16:13:47 2000
+++ gretl-0.96-new/gretl_sh.in	Mon Nov 12 14:47:34 2001
@@ -5,11 +5,12 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 bindir=@bindir@
+bindir_x=/usr/X11R6/bin
 
 if [ "x$DISPLAY" = "x" ] ; then
     exec ${bindir}/gretlcli "$@"
 else
-    exec ${bindir}/gretl_x11 "$@"
+    exec ${bindir_x}/gretl_x11 "$@"
 fi
 
 
